BFB/src/main/java/BFB/GUI/ForcePanel.java

```diff
@@ -93,7 +93,7 @@ private void initComponents() {
         jPanel2 = new javax.swing.JPanel();
         jLabel1 = new javax.swing.JLabel();
         txtForceName = new javax.swing.JTextField();
-        cmbForceType = new javax.swing.JComboBox();
+        cmbForceType = new javax.swing.JComboBox<String>();
```

Comment:
So I found a way to fix these in the form designer, which will allow the updates to persist when editing any dialogs in Netbeans. For example, open BFB/GUI/ForcePanel.java in Netbeans, right click the cmbJForceType component, go to Code, and then type `<String>` in the "Type Parameters" box and then save the form:

![typeproperties](https://user-images.githubusercontent.com/5447759/102571559-ae3d3c00-40e1-11eb-9a2c-d162ac30d56b.png)

Can you do these for any changes that take place in autogenerated code sections, instead of modifying the Java directly?

Reply:
You shouldn't need to revert or undo the previous changes, because once you open and save the forms in Netbeans the changes in the autogenerated code sections will be erased anyway

Reply:
Not using NetBeans. Using VS Code. Although that is good to know for future forms y'all mess with. :)

Reply:
Unfortunately Netbeans is required for any changes to GUI code :( I wish there was a way to make it totally IDE-agnostic but it would be too hard for us not to have a form designer.

I only use Netbeans for that purpose, any IDE will work for logic changes that don't happen inside of autogenerated GUI code